columnphysics/icepack_parameters.F90

```diff
@@ -162,6 +162,11 @@ module icepack_parameters
                                   ! 'linear_salt' = -depressT * sss
                                   ! 'mushy' conforms with ktherm=2
 
+      character(len=char_len), public :: &
+         saltflux_option  = '4psu'! Salt flux computation
```

Comment:
It would be better to add a separate parameter for the reference value itself, rather than hardwiring it for 4 psu.  What if some other model prefers 4.5 psu?  You could use prognostic and constant as the types for now, and then maybe someone would add some other type in the future (seasonal? stochastic?).

Also, "saltflux_option" sounds clunky to me, maybe because it's not immediately clear what kind of "option" it is.  We use "type" in other flags, e.g. bgc_flux_type.  The equivalent here would work:  salt_flux_type.  (And yes I realize that there's a tfrz_option, so this revision is optional...)